requirements.txt

```diff
@@ -0,0 +1,12 @@
+Django==2.1.2
```

Comment:
Do we need all these packages? Can You pick only these, that we actually going to use?

Reply:
There's no Django initialize, so maybe leave here only `Django`?